content/en/blog/_posts/2025-05-22-gateway-api-v1.3/index.md

```diff
@@ -0,0 +1,370 @@
+(and Hostname if supported by the protocol) in order for all Listeners to be
+*compatible* and not conflicted over which traffic they should receive.  
+Furthermore, implementations can *merge* separate Gateways into a single set of
+Listener addresses if all Listeners across all Gateways are compatible.  The
+management of merged Listeners is under-specified in releases prior to v1.3.0.
+
+With this feature, the specification on merging is expanded.  Implementations
+must treat the parent Gateways as having the merged list of all listeners from
+itself and from attached XListenerSets, and validation of this list of listeners
+must behave the same as if the list were part of a single `Gateway`. Within a single
+Gateway, Listeners will then be ordered using the following precedence:
+
+1. Single Listeners (not a part of an `XListenerSet`) first, followed by
+1. Listeners ordered by creation time (oldest first), followed by
+1. Listeners ordered alphabetically by “namespace/name”
+
+### XBackendTrafficPolicy (Retry Budgets)
+Leads: [Eric Bishop](https://github.com/ericbishop), [Mike Morris](https://github.com/mikemorris)
+
+GEP-3388: [Retry Budgets](https://gateway-api.sigs.k8s.io/geps/gep-3388)
+
+This feature specifies the configuration of a "retry budget" across all endpoints
+of a destination service.  This is used to limit additional client-side retries
+after reaching a configured threshold. The budget can be configured using a
+maximum percentage of active requests, or an interval during which requests will
+be considered. The development of this specification changed the existing
+experimental field `BackendLBPolicy` into `XBackendTrafficPolicy` in the interest
+of reducing the proliferation of policy resources that had commonalities.
+
+The following example shows an `XBackendTrafficPolicy` that applies a
+`retryConstraint` that represents a budget that limits the retries to a maximum
+of 20% of requests, over a duration of 10 seconds, and to a minimum of 3 retries
+over 1 second.
+
+```yaml
+apiVersion: gateway.networking.x-k8s.io/v1alpha1
+kind: XBackendTrafficPolicy
+metadata:
+  name: traffic-policy-example
+spec:
+  retryConstraint:
+    budget: 
+        percent: 20
+        interval: 10s
+    minRetryRate:
+      count: 3
+      interval: 1s
+    ...
+```
+
+## Try it out
+
+Unlike other Kubernetes APIs, you don't need to upgrade to the latest version of
+Kubernetes to get the latest version of Gateway API. As long as you're running
+Kubernetes 1.26 or later, you'll be able to get up and running with this version
+of Gateway API.
+
+To try out the API, follow our [Getting Started Guide](https://gateway-api.sigs.k8s.io/guides/).
+As of this writing, two implementations are already conformant with Gateway API
+v1.3. In alphabetical order:
+
+- [Cilium](https://github.com/cilium/cilium), Experimental channel
+- [Istio](https://istio.io), Experimental channel
```

Comment:
Envoy Gateway also submitted a conformance report - @Xunzhuo and/or @arkodg how can people try Gateway v1.3 with Envoy Gateway?

Reply:
I added a link to Envoy Gateway v1.4.0.